sdk/search/search-documents/test/internal/geographyPoint.spec.ts

```diff
@@ -7,11 +7,14 @@ import GeographyPoint from "../../src/geographyPoint";
 
 describe("geographyPoint", () => {
   it("JSON.stringify", () => {
-    const geoPoint = new GeographyPoint(47.669444, -122.123889);
+    const geoPoint = new GeographyPoint({
+      longitude: -122.123889,
+      latitude: 47.669444
+    });
     const result = JSON.parse(JSON.stringify(geoPoint));
     assert.deepEqual(result, {
       type: "Point",
-      coordinates: [-122.123889, 47.669444],
+      coordinates: [47.669444, -122.123889],
```

Comment:
the change on this line should be reverted right? Since longitude comes first in the encoding?

Reply:
Done